### ISSUE TITLE
A question about dataset partition

### ISSUE THREAD

**Issue by BEFLIPPED** [LMD0311/PointMamba#35]
I find some settings in `finetune_modelnet.yaml` and `ModelNet40.yaml`, as follows

in `finetune_modelnet.yaml`:
…
dataset: {
  train: { _base_: cfgs/dataset_configs/ModelNet40.yaml,
           others: { subset: 'train' } },
  val: { _base_: cfgs/dataset_configs/ModelNet40.yaml,
         others: { subset: 'test' } },
  test: { _base_: cfgs/dataset_configs/ModelNet40.yaml,
          others: { subset: 'test' } } }
…

in `ModelNet40.yaml`:
NAME: ModelNet
DATA_PATH: data/ModelNet/modelnet40_normal_resampled
N_POINTS: 8192
NUM_CATEGORY: 40
USE_NORMALS: FALSE

Does these settings mean that the validation set and the test set are the same data？
Thank you!

**Comment by LMD0311** [comment]
Yes, the dataset division aligns with other research.

**Comment by BEFLIPPED** [comment]
> Yes, the dataset division aligns with other research.

OK, Thank you.
